pkgs/tools/misc/memtest86-efi/default.nix

```
@@ -2,10 +2,10 @@
 
 stdenv.mkDerivation rec {
   pname = "memtest86-efi";
-  version = "8.2";
+  version = "8.3";
 
   src = fetchzip {
-    # TODO: The latest version of memtest86 is actually 8.2, but the
+    # TODO: The latest version of memtest86 is actually 8.3, but the
```

maybe @r-ryantm should ignore comments?
```suggestion
    # TODO: We're using the previous version of memtest86 because the
```

---

Good suggestion!

---

hard to know what should or shouldn't be changed. Generally i see the changing of comments to be correctly aligned with the version bumps